src/System.IO.Pipelines.Extensions/ReadableBufferSequence.cs

```diff
@@ -16,16 +15,16 @@ public ReadableBufferSequence(ReadOnlyBuffer buffer) : this()
             _buffer = buffer;
         }
 
-        public Collections.Sequences.Position First => Collections.Sequences.Position.Create(_buffer.Start.Segment, _buffer.Start.Index);
+        public Position Start => new Position(_buffer.Start.Segment, _buffer.Start.Index);
```

Comment:
Why have start at all if we treat `default` as a `Start`?

Reply:
I dont think we treat default as start. I remove this early this week.